config/routes.rb

```diff
@@ -755,7 +755,6 @@
         end
       end
 
-      resources :organisations, only: %i[index show], path: 'organisational-permissions'
```

Comment:
Discussed with @benswannack and we're not sure if we need a redirect here as we're removing a route

Reply:
I've added a redirect for this one, because it maps directly to a new one

Reply:
Can we add a `# TODO: Remove redirects from Oct 12 2021`? I think giving it a couple of months will be enough

Reply:
I'm not sure of the benefits of having only a temporary redirect here. We haven't sent any emails out with this URL in, so users will only be trying to go to the URL through a bookmark or via browser history. If a bookmark continues to work then a user will likely keep using it.

I think we should leave it in permanently. It doesn't do any harm to users, and it's self-documenting the history of the routes.

What do you think?

Reply:
Added a TODO to revisit this at a later date. If noone has been to the URL before Nov 2021 then we can probably just remove it